queries/futures/useGetCurrentPortfolioValue.ts

```diff
@@ -0,0 +1,61 @@
+import { useQuery, UseQueryOptions } from 'react-query';
+import { useRecoilValue } from 'recoil';
+import { utils as ethersUtils } from 'ethers';
+
+import { appReadyState } from 'store/app';
+import { isL2State, walletAddressState } from 'store/wallet';
+
+import Connector from 'containers/Connector';
+import QUERY_KEYS from 'constants/queryKeys';
+import { mapFuturesPosition } from './utils';
+import Wei, { wei } from '@synthetixio/wei';
+
+const useGetCurrentPortfolioValue = (
+	markets: string[] | [],
+	options?: UseQueryOptions<any | null>
+) => {
+	const isAppReady = useRecoilValue(appReadyState);
+	const isL2 = useRecoilValue(isL2State);
+	const walletAddress = useRecoilValue(walletAddressState);
+	const { synthetixjs } = Connector.useContainer();
+
+	return useQuery<any | null>(
+		QUERY_KEYS.Futures.Positions(markets || [], walletAddress || ''),
+		async () => {
+			const {
+				contracts: { FuturesMarketData },
+			} = synthetixjs!;
+			if (!markets) return null;
+
+			try {
+				const positionsForMarkets = await Promise.all(
+					(markets as [string]).map((market: string) =>
```

Comment:
Not sure if this is intentional, but I think this might be interpreted by Typescript as an array with one string. For multiple markets, I think we should do `string[]` instead of `[string]`.

Reply:
fixed